packages/react/src/tracking/use-track.ts

```diff
@@ -0,0 +1,29 @@
+import type { TrackingEventDetails } from '@openfeature/web-sdk';
+import { useCallback } from 'react';
+import { useOpenFeatureClient } from '../provider';
+
+export type Track = {
```

Comment:
Couldn't this type live in core?

Reply:
It's used only in React; it's the type of the exported track hook - it happens to look like the track method but it's not a method.

Reply:
I can do `track: Tracking['track'];` actually, since this function actually matches that type.